packages/xo-server-test/src/vm/index.spec.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env jest */
+
+import { mapValues } from 'lodash'
+
+import config from '../_config'
+import xo from '../_xoConnection'
+
+describe('vm', () => {
+  describe('.set() :', () => {
+    const PROPERTIES_TO_SET_TO_NULL = {
+      coresPerSocket: 1,
+      cpuCap: 1,
+    }
+    it('set properties to null', async () => {
```

Comment:
IMHO, we should make the tests based on issues referencing them, what do you think?

Reply:
good idea :+1:

Reply:
There is now an issue:  #4514

Reply:
Maybe reference the PR then.

Reply:
Can you please suggest a syntax to use in this case?